lib/package.gd

```diff
@@ -691,6 +691,31 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; subgroups package
```

Comment:
GAP builds fine but `make manuals` doesn't build.  Here is the output

    ( echo 'SaveWorkspace( "doc/wsp.g" );' | ./bin/gap-default64.sh -b -m 100m -o 750m -q -x 80 -r  )
    #I  GAP: needed package gapdoc cannot be loaded
    Error, failed to load needed package `gapdoc' (version >= 1.2) called from
    func(  
    ); on line 143 of file /Users/srm/Documents/sandeep/cst/dev/gap/lib/system.g \
    called from
    <function "CallAndInstallPostRestore">( <arguments> )
     called from read-eval loop at line 
    899 of /Users/srm/Documents/sandeep/cst/dev/gap/lib/init.g
    you can 'quit;' to quit to outer loop, or
    you can 'return;' to continue
    brk>